.configure

```diff
@@ -1,13 +1,18 @@
 {
   "project_name": "simplenote-ios",
-  "branch": "master",
-  "pinned_hash": "10c6ba49771ae477c2e9a7689d6fa808abab2fc6",
+  "branch": "simplenote-screenshot-user-test",
+  "pinned_hash": "72f07559c4624918965acf9eacde9b843c9edd26",
```

Comment:
Since I wasn't sure on what's the best approach to use for the secrets, I pushed my stuff to a branch.

Reply:
Yay! I think you're the first one to ever use the branch support I added to this tool 😄 

Feel free to just commit to `master` for mobile secrets – because everyone else's changes are pinned, it's unlikely you could break someone else's stuff – also, PRs don't work well with `git-crypt`, so it's easier just to use `master` 😄